module-types/albis-elcon/WANX-4SHDSL.yaml

```diff
@@ -0,0 +1,8 @@
+---
+manufacturer: albis-elcon
+model: WANX-4SHDSL
```

Comment:
According to the PDF you provided, it appears that part number `900420` is not `model: WANX-4SHDSL`, it is actually `model: WANX-4SHDSL-Extender`. Please update the model and file name accordingly

Reply:
It's described as being an extender in the datasheet but the model is printed without `-Extender` on the label.